### PR TITLE
Drop typings

### PR DESCRIPTION
**Pull request by kamilkisiela**
@stubailo

### CODE REVIEW

File: src/store.ts [hunk 1]
```diff
@@ -123,10 +123,12 @@ export function createApolloStore({
     enhancers.push(applyMiddleware(crashReporter));
   }
 
+  const [first, ...rest] = enhancers; // XXX see why it's needed
```
Comment (kamilkisiela) [inline src/store.ts]: Weird thing

File: tsconfig.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "outDir": "lib",
     "allowSyntheticDefaultImports": true,
     "removeComments": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```
Comment (kamilkisiela) [inline tsconfig.json]: I had to do this so `graphql-typings` package would work.

**Comment by kamilkisiela** [comment]
I will finish fixing type errors later, but if someone wants to help that would be great

**Comment by kamilkisiela** [comment]
@stubailo ready

**Comment by kamilkisiela** [comment]
We can wait for [`2ddfa4`](https://github.com/nitintutlani/typed-graphql/commit/2ddfa440c2690aa8c8f7473b659be63a2935aef3) to be published. This way we can keep `noImplicitAny` enabled.

**Comment by jbaxleyiii** [comment]
@kamilkisiela why are we doing this? And should I do the same for react-apollo? 👍

**Comment by kamilkisiela** [comment]
@jbaxleyiii [*"The Future of Declaration Files"*](https://blogs.msdn.microsoft.com/typescript/2016/06/15/the-future-of-declaration-files/)

It's way more convenient to use `npm` to install declarations than to have separate tools to do that.

With this approach all you have to do is to run `npm install`.

**Comment by jbaxleyiii** [comment]
@kamilkisiela  awesome! I'll do the same 🎉

File: tsconfig.json (hunk 2 above)
Comment (stubailo) [inline tsconfig.json]: Can we fix this? Then I can merge.

Comment (kamilkisiela) [inline tsconfig.json]: Latest changes of `graphl-typings` allows to enable `setImplicitAny`. It should be released soon, then I can fix it. I will @mention you when new version will be available.

**Pull request merged**